src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -21,8 +23,16 @@ public void actionPerformed(ActionEvent e) {
 			FiguresJoe.figureScript1(driverManager.getCurrentDriver());
 		}else if(action.equals("Figure Joe 2")){
 			FiguresJoe.figureScript2(driverManager.getCurrentDriver());
-		}else{//domyślny
-			FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+		}else if(action.equals("SQUARE")){
+			ComplexCommandFactory factory = new ComplexCommandFactory(driverManager.getCurrentDriver(),0,0);
+			ComplexCommand squareFigure = factory.createRectangle(120,120);
+			squareFigure.execute();
+
+		}else if(action.equals("TRIANGLE")){
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.